sentinelone/plugin.spec.yaml

```diff
@@ -1111,6 +1111,36 @@ actions:
         description: Number of entities affected by the requested operation
         type: integer
         required: false
+  blacklist:
+    title: Blacklist
```

Comment:
This might need to be more descriptive so it is easier to know what this action is used for up front.  Perhaps `Blacklist Hash`

Reply:
Fair point but we decided to do all the EDR integrations with "Blacklist" as title.

Reply:
@jschipp-r7 We should revisit this in the future.  I can see having a `Blacklist` (if it is for everything (hash, system name, etc) but it appears we have other actions that are also `Blacklist by`.  How would a customer know without already using that action?

In addition, how would a customer know they can remove from blacklist form this action?

Reply:
Based on most of the EDR vendors we looked at, "Blacklist" is often used to reference ban hashes. We're using "Quarantine" for endpoint isolation. We will be consolidating the other actions here so there's only a single Blacklist action.

The description of the action lets them know they can perform both.

Reply:
I agree though, it's not perfect